dataset/dictionary/characters.json5

```diff
@@ -1886,7 +1888,7 @@
   {
     en: "Scaramouche",
     ja: "スカラマシュ",
-    zhCN: "丑角",
+    zhCN: "斯卡拉姆齐",
     notes: "中国のプレイヤーの間では「スカラマシュ」(丑角) よりも「散兵」の名で呼ばれることが多い。",
```

Comment:
Sorry it was a mistake I made before. 

Also sorry for forgetting to reply the email about Harbingers as I was too busy to reply the it yesterday. I have understood your point of view and I agree that it would be better to keep 2 groups of translation for each harbinger. (Tartaglia <-> 达达利亚, Childe <->公子) And, for things about Scaramouche, it plays a similar role to Tartaglia, so I use the transliteration in Chinese.

Reply:
> Sorry it was a mistake I made before.
>
> Also sorry for forgetting to reply the email about Harbingers as I was too busy to reply the it yesterday.

No problem :)

> I have understood your point of view and I agree that it would be better to keep 2 groups of translation for each harbinger.

Thanks, let's do so.

> And, for things about Scaramouche, it plays a similar role to Tartaglia, so I use the transliteration in Chinese.

I searched screenshot for Scaramouche's first appearance in Chinese.
The fourth image in this article:
https://www.bilibili.com/read/mobile?id=13087010

And I understand what you mean. There is no official transliteration for the "name" (e.g. Scaramouche) in Chinese, right? What I called "name" is treated as just a translation for the codename.

If I understand correctly, let's delete Chinese translation from the "names" (Italian) and add note to explain that there is no official Chinese translation/transliteration.

Is 达达利亚 unofficial transliteration either? I sometimes look this word on the web.
Even if the word 达达利亚 does not appear in the game, I think we don't have to delete it if it is widely used in Chinese players. But if it is not widely used, let's delete it as well.

Because it may be a bit difficult to send patch to add notes using change request feature, I will add a commit in this PR. (Or I will create another PR.)
I feel a bit tired today, so let me do that tomorrow.

Reply:
Thanks for your investigation. However, in fact, all the transliteration I used came from the official. In the [official PV](https://www.bilibili.com/video/BV1mB4y1e7Lh?spm_id_from=333.337.search-card.all.click&vd_source=112a8fb7666254519c8ffd89584c016b) introducing the harbingers recently, the video introduction gave us the CV list and you can see the official transliteration:
<img width="350" alt="image" src="https://user-images.githubusercontent.com/48173350/179355864-7485b558-a7fc-482f-9f69-e947ec255f81.png">

Although Scaramouche didn't appear in the PV, I'm also sure that 斯卡拉姆齐 is an official name. You can see the subtitles of the PV at 2:07:
<img width="949" alt="image" src="https://user-images.githubusercontent.com/48173350/179355974-df16983a-273f-4896-9451-dddf5e6280cb.png">
(BTW, I'm also wondering why only the Chinese version has a subtitle 😂)

Regarding to 达达利亚, in fact it is the official name of the actionable character in the game. However, Chinese players really rarely call him 达达利亚 but 公子.
![IMG_7BFF43292167-1](https://user-images.githubusercontent.com/48173350/179356129-487c0d45-6543-4e16-bd9e-b88c1c106c12.jpeg)

Reply:
OK, I understood. Then let's keep these Chinese transliterations for the (Italian) names.

> (BTW, I'm also wondering why only the Chinese version has a subtitle 😂)

English version has subtitle too. But on YouTube, you need to press CC button to show it.
Japanese version only have useless auto-generated subtitle. Maybe miHoYo (Cognosphere) Japan employees are too busy...